hypothesis/via
# Integrate Checkmate as the blocking provider in Via

For: https://github.com/hypothesis/checkmate/issues/3

Testing notes

- Run `make dev`
- Goto: http://localhost:9080/proxy/https://bad.example.com/
- You should some error as the page isn't real
- Nothing shows in the logs for me, not sure why
- Start checkmate service with `make dev`
- Goto: http://localhost:9080/https://bad.example.com/
- You should now see the malicious blocking page
